### DISCUSSION TITLE
Stock Value Currency Euro instead of Dollar

### DISCUSSION THREAD

**Discussion by Fluxus00** [inventree/InvenTree#4565] [Q&A] [answered]
![stock_currency](https://user-images.githubusercontent.com/115626396/229466044-12b9485b-5b77-4bad-a883-2a2cee47da94.png)
Hey Guys,

as you can see in the example above I "purchased" all stock items with Euro as currency but the entire stock value still gets converted to Dollar. I already changed  the default currency to Euro in the config.yaml and the InvenTree settings. I do not want to manually convert the value back every time, because I only work with Euro as currency. So I was wondering if I just missed the key setting to solve my problem?
Thank you for the answers in advance!

**Comment by SchrodingersGat** [comment]
@Fluxus00 this should be displayed in your base currency. Can you please show a screenshot of your "currency settings" page?

**Comment by SchrodingersGat** [comment, marked as answer]
@Fluxus00 never mind, found there is a bug - fix incoming in https://github.com/inventree/InvenTree/pull/4567

**Reply by Fluxus00** [reply]
@SchrodingersGat Ah thank you very much, was already writing the response :D
